api/src/utils/default-user.ts

```diff
@@ -0,0 +1,58 @@
+// TODO: audit this object to find out which properties need to be updated.
+import { type Prisma } from '@prisma/client';
+
+export const defaultUser: Omit<Prisma.userCreateInput, 'email'> = {
+  about: '',
+  acceptedPrivacyTerms: false,
+  badges: {},
+  completedChallenges: [],
+  currentChallengeId: '',
+  emailVerified: true,
+  externalId: '',
+  is2018DataVisCert: false,
+  is2018FullStackCert: false,
+  isApisMicroservicesCert: false,
+  isBackEndCert: false,
+  isBanned: false,
+  isCheater: false,
+  isDataAnalysisPyCertV7: false,
+  isDataVisCert: false,
+  isDonating: false,
+  isFrontEndCert: false,
+  isFrontEndLibsCert: false,
+  isFullStackCert: false,
+  isHonest: false,
+  isInfosecCertV7: false,
+  isInfosecQaCert: false,
+  isJsAlgoDataStructCert: false,
+  isMachineLearningPyCertV7: false,
+  isQaCertV7: false,
+  isRelationalDatabaseCertV8: false,
+  isCollegeAlgebraPyCertV8: false,
+  isRespWebDesignCert: false,
+  isSciCompPyCertV7: false,
+  keyboardShortcuts: false,
+  location: '',
+  name: '',
+  unsubscribeId: '',
+  picture: '',
+  profileUI: {
+    isLocked: false,
+    showAbout: false,
+    showCerts: false,
+    showDonation: false,
+    showHeatMap: false,
+    showLocation: false,
+    showName: false,
+    showPoints: false,
+    showPortfolio: false,
+    showTimeLine: false
+  },
+  progressTimestamps: [],
+  // TODO: check what this is used for in api-server and if we need it
+  rand: 0,
```

Comment:
No, we do not need this. It was related to our days of saving passwords

Reply:
Okay, I'll create a PR removing it from the schema.